api/src/pcapi/routes/backoffice/templates/collective_offer/details_v2.html

```diff
@@ -135,7 +135,7 @@
   {% if has_permission("PRO_FRAUD_ACTIONS") %}
     {{ build_lazy_modal(url_for('backoffice_web.collective_offer.get_validate_collective_offer_form', collective_offer_id=collective_offer.id) , "validate-collective-offer-modal-" + collective_offer.id|string) }}
     {{ build_lazy_modal(url_for('backoffice_web.collective_offer.get_reject_collective_offer_form', collective_offer_id=collective_offer.id) , "reject-collective-offer-modal-" + collective_offer.id|string) }}
-    {% if move_offer_form %}
+    {% if is_feature_active("MOVE_OFFER_TEST") %}
```

Comment:
il vaudrait mieux continuer a s'assuerer que move_offer_form est présent non ?

```suggestion
    {% if is_feature_active("MOVE_OFFER_TEST") and move_offer_form %}
```

Reply:
Y'a un peu d'historique sur cette feature. Le  `move_offer_form` doit être supprimé, il n'est plus fourni par la vue mais initialisé au clique bouton.

Reply:
ok merci :)